src/app/editor-container/editor-container.component.scss

```diff
@@ -0,0 +1,3 @@
+.editor-parent-container {
+  overflow-x: hidden;
```

Comment:
There is an error with anyOf in the schemas so i have an error right now. @harunurhan is working on it. When is fixed i will check the preview pdf and try the anyOf support in the record editor.